components/Badge.js

```diff
@@ -37,7 +37,7 @@ export default function Badge({
   const badge = (
     <div
       title={title}
-      className={`absolute inline-block rotate-0 skew-x-0 skew-y-0 scale-x-100 scale-y-100 py-1 px-1.5 text-xs leading-none text-center whitespace-nowrap align-baseline font-bold ${
+      className={`absolute inline-block rotate-0 skew-x-0 skew-y-0 scale-x-100 scale-y-100 py-2 px-2 text-xs leading-none text-center whitespace-nowrap align-baseline font-bold ${
```

Comment:
This could be simplified to `p-2`

```suggestion
      className={`absolute inline-block rotate-0 skew-x-0 skew-y-0 scale-x-100 scale-y-100 p-2 text-xs leading-none text-center whitespace-nowrap align-baseline font-bold ${
```